common/eth2/src/lighthouse.rs

```diff
@@ -6,7 +6,8 @@ mod block_rewards;
 pub mod sync_state;
 
 use crate::{
-    BeaconNodeHttpClient, DepositData, Error, Hash256, Slot,
+    BeaconNodeHttpClient, DepositData, Hash256, Slot,
+    error::Error,
```

Comment:
Can revert this line
```suggestion
    BeaconNodeHttpClient, DepositData, Error,  Hash256, Slot,
```